config/credentials.yml.enc

```diff
@@ -1 +1 @@
-jbDbspan+ITIbLTnPw/+qjO6ZqFYa8gCSYEk8VBtdQSvRVPykc6iIStY6RcL5FzqpFjXDBj+22CNf/TaZxo/j8TyFHgw4FxW+0Cm7GX7OiyKSI3jGtZ+tiwlYXVX2St83ARaJHEfqbnxkqzWin/6XGXB+1HcFw+NSvHIFLNVc72NwahtfsKmldMh4nmWBnCeRdb3y4rgMZbICnkuinvpG10hM7ZsqUNW26SJWEucVtrW7UcLGjUHHCofPp2Upyv2ZFATUusfH57qRx8XCQ3jqtR0pfCqmAMoHyashWszHJtiSif73DC9Vl4cUoQas8oyMU8MML2jMseF3il+hsa7UwLtiUvtq0XCtBZagUJbLARD46p9CpvgaRGW2tHKzlB6n/J0yP2ZeEuk30TBXwsVK4aKj2mIJuLv6roz--FCZPUYZaij3fTfN6--9nqZakSLIXL63GjPUInInQ==
\ No newline at end of file
+ygtRHJ6yHeMdtb2NAbydKw7ouNlR0vSJmjY/U4sh057fMseAtkjerkopHHV0viDaNbWNyMlfech34SNiZzymG23DjM/VK658w1q6vWvPQB0FENwZeGzjItsl/aqtSoEJLUbFETKyhxUcVuFDGGfoyWvO/X7OxKwN1oZ8pxdg/1kGKvYEiyh3vjE33MuUjy6O2+mi/6jf7Xz1HAtlssNJNKHFDcr0QgGkbRjWB2q+CUTHmSh0jL89QFZuv7xSjJ5A55QmbOgYpxzmTV0IHE+cjjIm6AHhgq8KjOzFQz3VZZqTCdksxSeEUbUwSx3Q9GeUHwtUc3ded1yj+D3jofh1xZHehZORMWrrXAb01ygP6lwiK3gSZixOuJPbt8Ikfiikul6e9mVuTLrWmO1G4/Sqq0/dj4DGHRH1W7YCkpVII33P0COCkokUp11+UyAI1h3Ww1vKhntXlDDahVvHEI3dLbG0mVcGA6fdDF1fLWbBOFMkp5HFEtNRA0/6yG5jNGgDlVK0rjwm--xkVZ9L7UE/f+gY+W--VcrPY/ZLCsPBh0wyKQF4Ow==
```

Comment:
Please add info to README.md how to setup keys in dev mode.

Reply:
not use in dev mode